app/services/TrainAPI.rb

```diff
@@ -0,0 +1,12 @@
+class TrainAPI
+  include HTTParty
+
+  def initialize
+    self.class.base_uri 'localhost:3001'
```

Comment:
we wont want to commit this in this state using the hard coded localhost, i think some time looking at dotenv will be useful for this.
i also think this can be removed from the initializer and just be
```
base_uri ENV['TRAIN_API_URL']
```
once dotenv is loaded and configured